### DISCUSSION TITLE
App connect to self hosted

### DISCUSSION THREAD

**Discussion by JRansomed** [stoatchat/discussions#166] [Q&A] [answered]
Is there a way to connect the App to a self hosted server?  I can't seem to find it.

This project is looking amazing, thanks in advance!

**Comment by JRansomed** [comment]
Based on the silence, it would appear the app is not viable for private servers.  Or maybe it will be in the future.  Looked pretty promising, but seeing a maintainer reply to a more recent question and ignore this one seems to answer the question.  In case anyone else was wondering.

**Reply by insertish** [reply]
Hi, I started university two weeks ago, very busy, hoping to get through all open issues within the next couple of weeks, thanks. (Also if you're wondering I just happen to have seen this reply on our Git logs)

**Reply by JRansomed** [reply]
Awesome that you started university a couple weeks ago!  Definitely keep that the priority.  Please don't take my previous comments to heart.  I was attempting to kick the tires and test with family.  It seems that it's too early to test the app and the self hosted.  Nothing wrong with that.  Just didn't know if I was missing something.

You've created a project here that is going to get a lot of attention.  It's looking awesome!  Thanks for the work and time you've put into this.

**Comment by CadillacCoupeDeVille** [comment, marked as answer]
@JRansomed 
try
https://github.com/revoltchat/self-hosted/issues/7#issuecomment-937354346
